openedx/stanford/pavelib/utils/i18n_helpers.py

```diff
@@ -7,15 +7,138 @@
 import fnmatch
 import logging
 import os
+from path import Path
+import shutil
+import subprocess
 
 import django
 from django.conf import settings
+from i18n import segment
+from i18n.config import Configuration
 from i18n.execute import execute
+from paver.easy import sh
 import polib
 
+BABEL_COMMAND_TEMPLATE = (
+    "pybabel extract --mapping={config} "
+    "--add-comments='Translators:' --keyword='interpolate' "
+    "--output={output} {template_directory}"
+)
+BASE_DIR = Path('.').abspath()
+CONFIG = Configuration(filename=BASE_DIR / 'conf/locale/stanford_config.yaml')
 LOG = logging.getLogger(__name__)
 
 
+def clean_pofile(filename):
+    LOG.info("Cleaning %s", filename)
+    pofile = polib.pofile(filename)
+    fix_header(pofile)
+    fix_metadata(pofile)
+    pofile.save()
+
+
+def compile_js(language):
+    sh(django_cmd('lms', 'devstack', 'compilejsi18n', '-l', language))
```

Comment:
need an import for `django_cmd`: `from pavelib.utils.cmd import django_cmd`

Reply:
@kluo Moved in 49767b5a48cb0d6630ff1fd7ee0d5e78d647c1db